packages/sdk-react/src/utils/test-utils.tsx

```diff
@@ -211,6 +212,34 @@ export async function waitUntilTableIsLoaded(
   });
 }
 
+/**
+ * Waits for condition to be true
+ *
+ * @param predicate Predicate checking for condition
+ * @param timeout Wait function timeout
+ */
+export function waitForCondition(
+  predicate: () => boolean,
+  timeout: number = 1_000
+) {
+  return new Promise<void>((resolve, reject) => {
+    const interval = 50;
```

Comment:
Can we have this as passed argument with default value 50?

Reply:
Passed the check interval as a function parameter